### PR TITLE
DOC: Better docs for `jax.numpy`: `minimum` and `maximum`

### PR DESCRIPTION
**Pull request by rajasekharporeddy**
Part of #21461

Rendered Preview:
- https://jax--23469.org.readthedocs.build/en/23469/_autosummary/jax.numpy.minimum.html
- https://jax--23469.org.readthedocs.build/en/23469/_autosummary/jax.numpy.maximum.html

### CODE REVIEW

File: jax/_src/numpy/ufuncs.py [hunk 1]
```diff
@@ -753,14 +753,122 @@ def subtract(x: ArrayLike, y: ArrayLike, /) -> Array:
...
+    >>> x1 = jnp.array([[-2, 5, 7, 4],
+    ...                 [1, -6, 3, 8]])
+    >>> y1 = jnp.array([-5, 3, 6, 9])
+    >>> jnp.maximum(x1, y1)
+    Array([[-2,  5,  7,  9],
+           [ 1,  3,  6,  9]], dtype=int32)
+
+    Inputs having ``nan``:
+
+    >>> x2 = jnp.array([nan, -3, 9])
```
Comment (dfm) [inline jax/_src/numpy/ufuncs.py:865]: Looks like `nan` is undefined here!

**Review by dfm — COMMENTED** [review]
Thanks! There a small issue with the examples, but otherwise this looks good to me.

File: jax/_src/numpy/ufuncs.py (hunk 1 above)
Comment (rajasekharporeddy) [reply jax/_src/numpy/ufuncs.py:866]: Thanks! `nan` defined now.

**Review by dfm — APPROVED** [review]
Thanks!

**Pull request merged**